src/localization/pageInfoStrings.json

```diff
@@ -16,17 +16,20 @@
     "customer": "Customer",
     "entered_by": "Entered By",
     "entry_date": "Entry Date",
-    "months_stock_required": "Months Stock",
+    "months_stock_required": "Max MOS",
     "stocktake_name": "Stocktake Name",
     "supplier": "Supplier",
     "their_ref": "Their Ref",
     "by_batch": "By batch",
     "category": "Category",
-    "suggested_equals": "Our suggested quantity",
-    "suggested_formula": "DAILY CONSUMPTION x DAYS TO SUPPLY - STOCK ON HAND",
-    "amc_equals": "Average monthly consumption",
-    "amc_formula": "OUTGOING STOCK x NUMBER OF DAYS IN A MONTH / (DAYS IN PERIOD - DAYS OUT OF STOCK)",
-    "none_suggested": "None suggested"
+    "suggested_equals": "Suggested quantity",
+    "suggested_formula": "(Monthly use x Max MOS) - Their stock",
+    "amc_equals": "Monthly use",
+    "number_of_days_in_a_month": "Number of days in a month",
+    "number_of_days_in_a_month_formula": "= Number of days in a year / Number of months in a year",
+    "amc_formula": "(OUT x Number of days in a month) / (Days within period - DOS)",
```

Comment:
Adjusted these formulas slightly to refer back to the column names and added parentheseseis cause Rowena said its better with them